src/commands/testing/spec/testing.spec.server.ts

```diff
@@ -328,152 +97,392 @@ BRF:1
 BRH:1
 end_of_record`
 
-    const coverageLcov = await readFile(coverageLcovPath)
+  describe('SASVIYA', () => {
+    const target: Target = generateTarget()
+    const testUrl = (test: string) =>
+      `${target.serverUrl}/${
+        target.serverType === ServerType.SasViya
+          ? 'SASJobExecution'
+          : 'SASStoredProcess'
```

Comment:
is this VIYA specific?  If so I guess it doesn't need SASStoredProcess.  If it's generic, it should probably include the Server API route as well?